crates/bevy_render/src/camera/camera.rs

```diff
@@ -248,6 +248,25 @@ impl Camera {
         })
     }
 
+    /// Returns a 2D world position computed from a position on this [`Camera`]'s viewport.
+    ///
+    /// Useful for 2D cameras and other cameras with an orthographic projection pointing along the Z axis.
```

Comment:
Is it worth leaving stronger language? IIUC, the results are only correct when the projection is orthographic.

Reply:
I wonder if we should say something like

A more efficient version of [`viewport_to_world`] that only works for cameras with an orthographic projection, such as 2D cameras.